build.gradle

```diff
@@ -4,6 +4,8 @@ plugins {
 	id 'java-gradle-plugin'
 	id 'idea'
 	id 'eclipse'
+	id 'groovy'
+	id 'org.jetbrains.kotlin.jvm' version '1.3.31'
```

Comment:
uh, no need to grab extra groovy and kotlin just for this change

Reply:
Groovy is required for the tests in Spock. Kotlin was just a personal test I forgot to remove